pkg/server/skydb/authinfo.go

```diff
@@ -91,7 +92,26 @@ func (a AuthData) IsValid() bool {
 		}
 	}
 
-	return true
+	return !a.IsEmpty()
+}
+
+// IsEmpty would return true if
+//
+// 1. no entries or
+// 2. all values are null
+func (a AuthData) IsEmpty() bool {
+	emptyCount := 0
```

Comment:
I would check, if the len(a) == 0 exit first
then exit immediately if you find a non empty field.